docs/details/query-analytics.md

```diff
@@ -0,0 +1,88 @@
+# Query analytics under the hood
+
+## Components
+1. [Filters Panel](../get-started/./query-analytics.md#filters-panel)
+2. [Overview Panel](../get-started./query-analytics.md#overview-panel)
+3. [Details Panel](../get-started./query-analytics.md#details-panel)
+
+## How QAN collects data?
+- data are collected every minute (at 0 seconds, like 8:15:00, next in 8:16:00 etc)
+- collected data are represented by buckets
+
+## What is a bucket/How is it created?
+- buckets contains all data captured during one minute interval
+- once a bucket is created it is sent to PMM Server where it is parsed and saved in the clickhouse database. All QAN related data is stored there. Clickhouse is part of PMM Server, or you can use an external one
+- queries in buckets are aggregated by query ID. It means one row in list overview for all queries with same query ID
+- query IDs are calculated different depends on technology and query source  
+**MySQL**   
+DIGEST (Perfschema) and fingerprint (Slowlog) are same. It is query without sensitive data. In case below both queries will have **same query ID**  
```

Comment:
```suggestion
For MySQL, the query ID is based on the DIGEST value (for Performance Schema) or the MD5 checksum of the fingerprint (for slow log).

DIGEST (Perfschema) and fingerprint (Slowlog) are same. It is query without sensitive data. In case below both queries will have **same query ID**. The DIGEST value is generated from the query text without sensitive data.
```